index.html

```diff
@@ -985,93 +985,49 @@ <h4>Deletion</h4>
                     <section id="exploration-directory-api-registration-listing" class="normative">
                         <h4>Listing</h4>
                         <p>
-                            <span class="rfc2119-assertion" id="tdd-reg-list">
+                            The listing endpoint provides a way to query the collection of TD objects
+                            from the directory. The Search API may be used to retrieve <a>Partial TD</a>s
+                            or TD fragments; see [[[#exploration-directory-api-search]]].
+                        </p>
+                        
+                            
+                        <p>
+                            <span class="rfc2119-assertion" id="tdd-reg-list-method">
```

Comment:
Assertions should ideally be self-contained.  I would add "from the retrieveTDs property".  BTW, "retrieveTDs" is a verb (phrase).  Properties should ideally be nouns, and actions verbs.  This is a bikeshed issue, but maybe something like "allTDs" makes more sense.   Likewise, it seems query actions like "searchJSONPath" are properties... that doesn't necessarily make sense either.

Reply:
BTW, we don't have to fix this in this PR, but maybe we should create another issue to clean this up.   We should probably create an actual TM also.

Reply:
Note this point is being discussed in #133